templates/AngularSpa/ClientApp/boot.server.ts

```diff
@@ -17,12 +17,12 @@ export default createServerRenderer(params => {
     ];
 
     return platformDynamicServer(providers).bootstrapModule(AppModule).then(moduleRef => {
-        const appRef = moduleRef.injector.get(ApplicationRef);
+        const appRef: ApplicationRef = moduleRef.injector.get(ApplicationRef);
```

Comment:
Probably because `ApplicationRef` is an abstract class this type wasn't inferred automatically.